devel/ci/integration/tests/test_bodhi.py

```diff
@@ -507,3 +508,113 @@ def test_get_overrides_view(bodhi_container, db_container):
         with read_file(bodhi_container, "/httpdir/errorlog") as log:
             print(log.read())
         raise
+
+
+def test_get_build_json(bodhi_container, db_container):
+    """Test ``/builds/{nvr}`` path"""
+    # Fetch build(of latest update) from DB
+    query_updates = (
+        "SELECT "
+        "  id "
+        "FROM updates "
+        "ORDER BY date_submitted DESC LIMIT 1"
+    )
+    query_builds = (
+        "SELECT "
+        "  nvr, "
+        "  release_id, "
+        "  signed, "
+        "  type, "
+        "  epoch "
+        "FROM builds "
+        "WHERE update_id = %s LIMIT 1"
+    )
+    db_ip = db_container.get_IPv4s()[0]
+    conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
+    with conn:
+        with conn.cursor() as curs:
+            curs.execute(query_updates)
+            update_id = curs.fetchone()[0]
+            curs.execute(query_builds, (update_id, ))
+            row = curs.fetchone()
+            nvr = row[0]
+            release_id = row[1]
+            signed = row[2]
+            type = row[3]
+            epoch = row[4]
```

Comment:
You could probably express all of the above with one line:

```
nvr, release_id, signed, type, epoch = curs.fetchone()
```

Also, you might want to use a different variable name than ```type```, since that's a Python keyword.